phpunit.xml.dist

```diff
@@ -15,11 +15,11 @@
         convertNoticesToExceptions="true"
         convertWarningsToExceptions="true"
         processIsolation="false"
-        stopOnError="false"
-        stopOnFailure="false"
+        stopOnError="true"
+        stopOnFailure="true"
```

Comment:
Is it needed to change this policy?

The downside of this change if I'm not mistaken would be that we will not get the full report of the different tests that are failing. What are the benefits of this change?

Thanks!

Reply:
Yup,
without those fixes phpunit always exit with "0". Normally this is an acceptable behavior, but if I need prevent push by checking exit code, I need to impose onFailure and onError like the before snippet.

But feel free to suggest me the best approach do you prefer! :)

Reply:
A further consideration: instead to change dist file, I can improve phpunit like that:
`phpunit --stop-on-failure --stop-on-error [...]`

Reply:
Makes sense. Thanks for the clarification 🙂

Yes, I would vote for leaving the `stopOnError="false"` and `stopOnFailure="false"` config options in the `phpunit.xml.dist`. Why:

* This way we would be having a default config for PhpUnit useful for the majority of the use cases (locally developing and being interested in all the failing test due to our changes)
* We can cover the minority of the use cases (running CI pipeline or `pre-push` hooks) with the second approach you've proposed (running PhpUnit with the command line flags)

Do you agree? What do you think?

Reply:
Yep,

I do agree with you, I consider prepush like so a "safe guard" of pushing new code. Usually, during code session, I tend to run singular test more. After when I'm confortable with my code I start to run all test suite...

Pheew! Do you think this PR can be merged?  👼